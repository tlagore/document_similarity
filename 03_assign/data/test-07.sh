#!/usr/bin/env bash
sbt --error 'set showSuccess := false' 'run data/articles_10000.txt  " " 0.9 3 100 4 false true false'
