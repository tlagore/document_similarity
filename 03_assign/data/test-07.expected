Computing similarity with parameters: 
    Filename: data/articles_10000.txt
    Separator: [ ]
    Minimum similarity: 0.9
    Shingle size: 3
    Hash count: 100
    Band size: 4
    Compute Jaccard similarity: false
    Compute all minHashes similarity: true
    Print hash coefficients: false

Pair                     Jaccard              minHash                  lsh
--------------------------------------------------------------------------
t1012,t9363                  ---              1.00000              1.00000
t104,t4172                   ---              1.00000              1.00000
t1057,t5702                  ---              1.00000              1.00000
t1088,t5015                  ---              0.98000              0.92000
t1142,t2475                  ---              0.96000                  ---
t1206,t4591                  ---              0.98000              0.92000
t1295,t6680                  ---              1.00000              1.00000
t1297,t4638                  ---              0.98000              0.92000
t1374,t3257                  ---              1.00000              1.00000
t1403,t5999                  ---              1.00000              1.00000
t1436,t492                   ---              1.00000              1.00000
t1488,t9549                  ---              0.96000                  ---
t1513,t764                   ---              1.00000              1.00000
t1583,t9230                  ---              1.00000              1.00000
t1621,t7958                  ---              1.00000              1.00000
t1726,t9170                  ---              1.00000              1.00000
t1768,t5248                  ---              0.99000              0.96000
t1782,t7716                  ---              1.00000              1.00000
t1898,t8090                  ---              1.00000              1.00000
t1952,t3495                  ---              0.96000                  ---
t1998,t5871                  ---              1.00000              1.00000
t2001,t5239                  ---              1.00000              1.00000
t2023,t980                   ---              0.98000              0.92000
t2100,t6571                  ---              1.00000              1.00000
t2121,t3020                  ---              1.00000              1.00000
t2356,t4015                  ---              1.00000              1.00000
t2390,t4969                  ---              1.00000              1.00000
t2432,t673                   ---              1.00000              1.00000
t2535,t8642                  ---              0.99000              0.96000
t269,t8413                   ---              1.00000              1.00000
t2839,t9303                  ---              0.97000                  ---
t288,t6999                   ---              1.00000              1.00000
t2957,t7111                  ---              0.95000                  ---
t3072,t7923                  ---              1.00000              1.00000
t3088,t797                   ---              1.00000              1.00000
t3136,t8469                  ---              1.00000              1.00000
t321,t6539                   ---              1.00000              1.00000
t3268,t7998                  ---              0.96000                  ---
t3358,t4022                  ---              1.00000              1.00000
t3446,t379                   ---              1.00000              1.00000
t3466,t7563                  ---              1.00000              1.00000
t3575,t8979                  ---              0.96000                  ---
t3600,t644                   ---              1.00000              1.00000
t3684,t617                   ---              1.00000              1.00000
t3702,t6235                  ---              1.00000              1.00000
t3725,t4099                  ---              0.99000              0.96000
t3727,t3982                  ---              1.00000              1.00000
t3783,t6092                  ---              1.00000              1.00000
t3889,t538                   ---              1.00000              1.00000
t4211,t9248                  ---              1.00000              1.00000
t4455,t7717                  ---              1.00000              1.00000
t4467,t6205                  ---              1.00000              1.00000
t448,t8535                   ---              1.00000              1.00000
t4530,t7907                  ---              0.98000              0.92000
t4615,t8496                  ---              1.00000              1.00000
t462,t7069                   ---              1.00000              1.00000
t4628,t646                   ---              1.00000              1.00000
t4755,t5544                  ---              1.00000              1.00000
t4792,t7973                  ---              1.00000              1.00000
t4910,t5780                  ---              1.00000              1.00000
t5262,t6261                  ---              1.00000              1.00000
t5304,t7320                  ---              0.99000              0.96000
t5411,t9894                  ---              1.00000              1.00000
t5416,t9355                  ---              1.00000              1.00000
t5442,t906                   ---              1.00000              1.00000
t5551,t7693                  ---              0.99000              0.96000
t6244,t969                   ---              0.99000              0.96000
t6370,t9445                  ---              1.00000              1.00000
t6520,t6906                  ---              1.00000              1.00000
t6613,t9385                  ---              1.00000              1.00000
t7270,t8387                  ---              0.96000                  ---
t7412,t7623                  ---              0.95000                  ---
t7527,t8101                  ---              0.99000              0.96000
t787,t9596                   ---              0.97000                  ---
t8164,t9455                  ---              1.00000              1.00000
t8306,t8805                  ---              0.98000              0.92000
t840,t9579                   ---              0.99000              0.96000
t8561,t9620                  ---              1.00000              1.00000
t8826,t9285                  ---              0.98000              0.92000
t8861,t9724                  ---              1.00000              1.00000
--------------------------------------------------------------------------
